src/config/dynamic-theme-fixes.config

```diff
@@ -24834,6 +24834,15 @@ INVERT
 
 ================================
 
+whatruns.com
+
+INVERT
+a.navbar-brand img
+.tech-list.container div a img
```

Comment:
The all logo invert looks strange a little bit
why not using img[title="Buffer"] to invert one black icon?